src/ocamlorg_frontend/pages/books.eml

```diff
@@ -33,34 +46,30 @@ Learn_layout.render
 ~left_sidebar_html:None
 ~right_sidebar_html:None
 ~current:Books @@
-<div class="bg-default dark:bg-dark-default">
-    <div class="container-fluid"  x-data="{ language: 'english' }">
-        <div class="flex justify-between mb-0 lg:flex-row flex-col items-center">
-            <h2 class="font-bold mt-10 lg:mt-0 mb-10 lg:mb-0">15+ OCaml Books</h2>
-            <div class="flex justify-between flex-col lg:flex-row lg:space-x-6 space-y-5 lg:space-y-0 md:space-y-5 w-full lg:w-auto">
-                <%s! Forms.select
-                    ~attrs:{|name="Languages" x-model="language"|}
-                    ~options:[("english", "English"); ("french", "French"); ("italian", "Italian"); ("portugese", "Portugese"); ("japanese", "Japanese"); ("chinese", "Chinese")]
-                    "h-14" %>
-            </div>
-        </div>
-        <div :class="language != 'english' ? 'hidden' : 'block'">
-            <%s! render_books (List.filter (fun x -> x.Data.Book.language = "english") books) %>
-        </div>
-        <div :class="language != 'french' ? 'hidden' : 'block'">
-            <%s! render_books (List.filter (fun x -> x.Data.Book.language = "french") books) %>
-        </div>
-        <div :class="language != 'italian' ? 'hidden' : 'block'">
-            <%s! render_books (List.filter (fun x -> x.Data.Book.language = "italian") books) %>
-        </div>
-        <div :class="language != 'portugese' ? 'hidden' : 'block'">
-            <%s! render_books (List.filter (fun x -> x.Data.Book.language = "portugese") books) %>
-        </div>
-        <div :class="language != 'japanese' ? 'hidden' : 'block'">
-            <%s! render_books (List.filter (fun x -> x.Data.Book.language = "japanese") books) %>
-        </div>
-        <div :class="language != 'chinese' ? 'hidden' : 'block'">
-            <%s! render_books (List.filter (fun x -> x.Data.Book.language = "chinese") books) %>
+<div class="bg-default dark:bg-dark-default"> 
+    <h2 class="font-bold mt-10 mb-4 md:mb-4 lg:mt-0 mb-10">Books On OCaml (15)</h2>  
```

Comment:
```suggestion
    <h2 class="font-bold mt-10 mb-6 lg:mt-0">Books On OCaml (15)</h2>  
```

I see that you took the value from Figma inspect with developer mode. But the problem was that the text element in Figma also had whitespace within itself. There is a way to automatically shrink Figma text elements: double click on their bottom border.